tests/unittests/test_ds_identify.py

```diff
@@ -2120,4 +2140,42 @@ def _print_run_output(rc, out, err, cfg, files):
             P_SYS_VENDOR: "3DS Outscale\n",
         },
     },
+    "Not-WSL": {
+        "ds": "WSL",
+        "mocks": [
+            MOCK_VIRT_IS_KVM,
+        ],
+    },
+    "WSL-no-host-mounts": {
+        "ds": "WSL",
+        "mocks": [
+            MOCK_VIRT_IS_WSL,
+            MOCK_UNAME_IS_WSL,
+        ],
+        "files": {
+            "proc/mounts": (
+                "/dev/sdd / ext4 rw,errors=remount-ro,data=ordered 0 0\n"
+                "cgroup2 /sys/fs/cgroup cgroup2 rw,nosuid,nodev,noexec0 0\n"
+                "snapfuse /snap/core22/1033 fuse.snapfuse ro,nodev,user_id=0,"
+                "group_id=0,allow_other 0 0"
+            ),
+        },
+    },
+    "WSL-supported": {
+        "ds": "WSL",
+        "mocks": [
+            MOCK_VIRT_IS_WSL,
+            MOCK_UNAME_IS_WSL,
+        ],
+        "files": {
+            "proc/mounts": (
+                "/dev/sdd / ext4 rw,errors=remount-ro,data=ordered 0 0\n"
+                "cgroup2 /sys/fs/cgroup cgroup2 rw,nosuid,nodev,noexec0 0\n"
+                "C:\\134 /mnt/c 9p rw,dirsync,aname=drvfs;path=C:\\;uid=0;\n"
```

Comment:
Will the line in `/proc/mounts` always match `aname=drvfs`?

Reply:
Indeed it always match. The device name (the first field in `/proc/mounts`) changed in December (it used to be `drvfs` but it's now `LETTER:\\NUMBER`), thus we cannot rely on the device name. Yet across many versions of WSL I could check this part `aname=drvfs` is stable. `path=` is always there too, but I'd be skeptical about how stable that metadata position is to match it via regex in a single match.

I'd say `'^\S* \S* 9p \S*aname=drvfs;.*'` is a strong enough filter. That only matches entry where the third field is '9p' and the fourth contains 'aname=drvfs', where fields are separated by whitespaces.